src/pages/searchOperators.tsx

```diff
@@ -0,0 +1,162 @@
+import React, { ReactElement } from 'react';
+import TwoThirdsLayout from '../layout/Layout';
+import { CustomAppProps, ErrorInfo, NextPageContextWithSession } from '../interfaces';
+import CsrfForm from '../components/CsrfForm';
+import ErrorSummary from '../components/ErrorSummary';
+import FormElementWrapper from '../components/FormElementWrapper';
+import { getSessionAttribute } from '../utils/sessions';
+import { SEARCH_OPERATOR_ATTRIBUTE } from '../constants';
+import { isSearchOperatorAttributeWithErrors } from '../interfaces/typeGuards';
+import { getSearchOperators, OperatorNameType } from '../data/auroradb';
+import { getAndValidateNoc } from '../utils';
+
+const title = 'Search Operators - Fares Data Build Tool';
+const description = 'Search Operators page for the Fares Data Build Tool';
+
+type SearchOperatorProps = {
+    errors: ErrorInfo[];
+    searchText: string;
+    operators: OperatorNameType[];
+};
+
+const SearchOperators = ({
+    errors = [],
+    csrfToken,
+    operators,
+    searchText,
+}: SearchOperatorProps & CustomAppProps): ReactElement => (
+    <TwoThirdsLayout title={title} description={description}>
+        <ErrorSummary errors={errors} />
+        <CsrfForm action="/api/searchOperators" method="post" csrfToken={csrfToken}>
+            <>
+                <div className="govuk-form-group">
+                    <h1 className="govuk-label-wrapper">
+                        <label className="govuk-label govuk-label--l" htmlFor="event-name">
+                            Search for the operators that the ticket covers
```

Comment:
I think we're missing either some hint text or a label for the search bar - according to the design, it has Operator Name above the search box.